charts/linkerd2/README.md

```diff
@@ -84,32 +84,23 @@ The following table lists the configurable parameters of the Linkerd2 chart and
+| `global.createdByAnnotation`                 | Annotation label for the proxy create. Do not edit.                                                                                                                                   | `linkerd.io/created-by`              |
 | `global.identityTrustAnchorsPEM`            | Trust root certificate (ECDSA). It must be provided during install.                                                                                                                                                         ||
 | `global.identityTrustDomain`                | Trust domain used for identity                                                                                                                                                        | `cluster.local`                      |
-| `grafanaImage`                        | Docker image for the Grafana container                                                                                                                                                | `gcr.io/linkerd-io/grafana`          |
-| `disableHeartBeat`                    | Set to true to not start the heartbeat cronjob                                                                                                                                        | `false`                              |
-| `heartbeatSchedule`                   | Config for the heartbeat cronjob                                                                                                                                                      | `0 0 * * *`                          |
-| `prometheusImage`                     | Docker image for the Prometheus container                                                                                                                                             | `prom/prometheus:v2.11.1`            |
-| `prometheusLogLevel`                  | Log level for Prometheus                                                                                                                                                              | `info`                               |
+| `global.imagePullPolicy`                     | Docker image pull policy                                                                                                                                                              | `IfNotPresent`                       |
+| `global.linkerdNamespaceLabel`               | Control plane label. Do not edit                                                                                                                                                      | `linkerd.io/control-plane-component` |
+| `global.linkerdVersion`                      | Control plane version                                                                                                                                                                 | `stable-2.5.0`                       |
```

Comment:
@Pothulapati you may not have your branch up to date; I think @adleong already updated this version to "latest version" instead of a static one.

Reply:
Thanks, Good find :)